tests/tests_old.rs

```diff
@@ -1145,6 +1196,36 @@ fn test_init_with_configuration() {
     assert_eq!(config.lock().unwrap().all_targets, false);
 }
 
+#[test]
+fn test_init_with_configuration() {
+    test_init_impl::<IdentityConverter>();
+}
+
+#[test]
+fn test_init_with_configuration_camel_case() {
```

Comment:
While it's cool that it works for other case styles, I think we mostly want to support camelCase and kebab-case so I think this might give off a weird impression of what's supported and is a bit noisy.
Could only keep the tests for those 2 cases?

Reply:
We should keep a test for normal (snake case), kebab case and camel case. Kebab case is the most important alternative case because it's used in cargo config